docs/howto/use-with-samba.md

```diff
@@ -0,0 +1,239 @@
+   at `/srv/samba/alice`. It is visible to users on the network (`browsable`),
+   allows writing (`writable`), and restricts access to the `alice` user (`valid
+   users`).
+
+4. **Create a Samba User for `alice`:**
+   Add the `alice` user to the Samba database and set a password:
+
+   ```bash
+   sudo smbpasswd -a alice
+   ```
+
+   Follow the prompts to set the Samba password for the user.
+
+5. **Restart Samba Service:**
+   Restart the Samba service to apply the changes:
+
+   ```bash
+   sudo systemctl restart smbd
+   ```
+
+---
+
+### Steps for the Client
+
+1. **Install Samba Client:**
+   Install the required packages for connecting to Samba shares:
+
+   ```bash
+   sudo apt update
+   sudo apt install smbclient cifs-utils
+   ```
+
+2. **Test Access to the Share:**
+   Test connectivity using `smbclient` (replace `$SERVER` with the Samba
+   server's hostname or IP address):
+
+   ```bash
+   smbclient //$SERVER/alice -U alice
+   ```
+
+   Enter the Samba password for `alice` when prompted. If successful, a `smb: \>`
+   prompt appears.
+
+3. **Mount the Share:**
+   Create a mount point for the share:
+
+   ```bash
+   mkdir -p /home/alice/samba
+   ```
+
+   Mount the share using the `cifs` filesystem type:
+
+   ```bash
+   sudo mount -t cifs //$SERVER/alice /home/alice/samba -o user=alice,uid=$(id -u alice),gid=$(id -g alice)
+   ```
+
+   Enter the Samba password for `alice` when prompted.
+
+4. **Optional: Add to `/etc/fstab` for Persistent Mounting:**
+   To automatically mount the share at boot, use a credentials file:
+
+   - Create a credentials file:
+
+     ```bash
+     sudo editor /etc/samba/credentials
+     ```
+
+     Add the following lines:
+
+     ```
+     username=alice
+     password=YOUR_PASSWORD
+     ```
+
+   - Secure the credentials file:
+
+     ```bash
+     sudo chmod 600 /etc/samba/credentials
+     ```
+
+   - Update `/etc/fstab`:
+
+     ```
+     //$SERVER/alice /home/alice/samba cifs credentials=/etc/samba/credentials,uid=alice,gid=alice 0 0
+     ```
+
+5. **Verify the Mount:**
+   As the user `alice`, try accessing the shared directory:
+
+   ```bash
+   ls -la /home/alice/samba
+   ```
+
+   Verify write access by creating a test file:
+
+   ```bash
+   touch /home/alice/samba/test
+   ```
+
+   **Security Note:** Files and directories in the share may appear as owned by
+   `alice` on the client, but access control is enforced by the server. For
+   example:
+
+   - If `alice` does not have permission on the server, access will be denied
+     even if ownership appears correct on the client.
+
+   Example:
+
+   - Create a restricted directory on the server:
+
+     ```bash
+     sudo mkdir /srv/samba/alice/secrets
+     sudo chmod 700 /srv/samba/alice/secrets
+     ```
+
+   - Attempt to access it on the client:
+
+     ```bash
+     ls /home/alice/samba/secrets
+     ```
+
+     **Result:** *Permission denied.*
```

Comment:
Is "permission denied" the actual terminal output?

If so, it could be put in a codeblock, i.e.:

"The terminal output will indicate that the user does not have access to the restricted directory:

<replace-with-triple-backticks>text
Permission denied
<replace-with-triple-backticks>

Reply:
done